workflow/README.md

```diff
@@ -15,4 +15,5 @@ We provide Python and R versions of the images. Here are the images that are ava
     - [workflow-r-ds-pack-v1](./r/ds-pack/v1/) 
 
 ## Important
-- Every workflow image must set up a non-root user. Details for the same can be found [here](../knowledge-base/non-root-user.md).
\ No newline at end of file
+All Workflows must be set up as a non-root user.
+For more details, see [here](../knowledge-base/non-root-user.md).
```

Comment:
```suggestion
For more details, find/check 🤔 [here](../knowledge-base/non-root-user.md).
```

Reply:
This should read:

For more details, see [Non root users and Peak](../knowledge-base/non-root-user.md)

The non-root-user.md file also needs a few changes made to it still.
I'll do this and pass it over for a PR.

Reply:
I've updated the non-root-user.md file in Confluence:

[Non root users and Peak](https://peak-bi.atlassian.net/wiki/spaces/KBS/pages/3206316189/platform-global-images+readme+files#Non-root-users-and-Peak)

Reply:
Updated.